p2p/transport/websocket/websocket_test.go

```diff
@@ -36,10 +36,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func makeMsTransport() tptu.MsTransport {
+	muxer := tptu.NewMsTransport()
+	muxer.AddMuxer("/yamux/1.0.0", yamux.DefaultTransport)
+	return muxer
+}
+
 func newUpgrader(t *testing.T) (peer.ID, transport.Upgrader) {
 	t.Helper()
 	id, m := newInsecureMuxer(t)
-	u, err := tptu.New(m, yamux.DefaultTransport)
+	u, err := tptu.New(m, makeMsTransport())
```

Comment:
This is different than the old behavior. Why this change?

Reply:
that's because of the change of the way upgrader is constructed, the final product is the upgrader, so that the end result is the same. The existing code makes the multistream.Transport conform to network.Multiplexer interface so you can pass a yamux to upgrader, but that's not the case anymore after the multistream.Transport is removed.